llvm/lib/Object/ELF.cpp

```diff
@@ -10,6 +10,7 @@
 #include "llvm/ADT/StringExtras.h"
 #include "llvm/BinaryFormat/ELF.h"
 #include "llvm/Support/DataExtractor.h"
+#include <type_traits>
```

Comment:
Did this sneak back in in your latest round of changes?

Reply:
Yep